packages/prettier-plugin-java/src/printers/classes.js

```diff
@@ -88,63 +97,98 @@ class ClassesPrettierVisitor {
   }
 
   fieldDeclaration(ctx) {
-    return "fieldDeclaration";
+    const fieldModifiers = this.mapVisit(ctx.fieldModifier);
+    const unannType = this.visit(ctx.unannType);
+    const variableDeclaratorList = this.visit(ctx.variableDeclaratorList);
+
+    return join("", [
```

Comment:
- why join with empty string?
  - just concat instead in those situations?
- why explicitly specify whitespace?